packages/esm-patient-medications-app/src/medications-summary/order-basket-action-button.component.tsx

```diff
@@ -26,6 +26,7 @@ const OrderBasketActionButton: React.FC = () => {
         onClick={launchOrdersWorkspace}
       >
         <div className={styles.elementContainer}>
+          {pendingOrders && <Tag className={styles.countTag}>!</Tag>}
```

Comment:
Also I thought of a scenario where we have one saved form and an incomplete one, so which icon should we show, I would guess the one for a unsaved form since that is more crucial, so inorder to achieve that, you would need to have those two conditions follow each other like this.
```js
{pendingOrders ? (
            <Tag className={styles.countTag}>!</Tag>
          ) : items?.length > 0 ? (
            <Tag className={styles.countTag}>{items?.length}</Tag>
          ) : null}
```